index.html

```diff
@@ -1,16 +1,115 @@
-<!doctype html>
+<!DOCTYPE html>
 
 <html lang="en">
-<head>
-  <meta charset="utf-8">
-  <title>Great Idea - Responsive I</title>
+  <head>
+	<meta charset="utf-8" />
+	<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+	<title>Great Idea - Responsive I</title>
 
-  <link href="https://fonts.googleapis.com/css?family=Bangers|Titillium+Web" rel="stylesheet">
-  <link rel="stylesheet" href="css/index.css">
+	<link
+	  href="https://fonts.googleapis.com/css?family=Bangers|Titillium+Web"
+	  rel="stylesheet"
+	/>
+	<link rel="stylesheet" href="css/index.css" />
+  </head>
 
-</head>
-
-<body>
-<!-- Use past code you created here for Great Idea Web Page or past solution for Great Idea Web Page HTML here! -->
-</body>
-</html>
\ No newline at end of file
+  <body>
+	<!-- Container -->
+	<div class="container">
+	  <!-- Navbar -->
+	  <div class="navbar">
+		<nav>
+		  <a href="#">Services</a>
+		  <a href="#">Product</a>
+		  <a href="#">Vision</a>
+		  <a href="#">Features</a>
+		  <a href="#">About</a>
+		  <a href="#">Contact</a>
+		</nav>
+		<div class="logo"><img src="/img/logo.png" alt="" class="" /></div>
+	  </div>
+	  <!-- End Navbar -->
+	  <!-- Hero -->
+	  <div class="hero">
+		<div class="hero-content">
+		  <h1>Innovation<br />on<br />demand</h1>
+		  <button>Get Started</button>
+		</div>
+		<div class="hero-img">
+		  <img src="/img/header-img.png" alt="" />
```

Comment:
It'd be better to use relative positioning here (and it has a bit more compatibility with other sites): `img/header-img.png` (without the starting `/`).

Reply:
Alright cool! Still trying to get the grasp of that, but it makes sense.